src/java/voldemort/utils/MetadataVersionStoreUtils.java

```diff
@@ -80,7 +80,12 @@ public static Properties mergeVersions(Properties prop1, Properties prop2) {
             return prop1;
         }
 
-        Properties result = new Properties(prop1);
```

Comment:
I believe Props we are using mostly from the Config perspective. We are using the Properties for  our versions serializing to byte array and loading back from byte array. marking mergeVersions as deprecated is not an option without the alternative.